src/poprox_concepts/api/recommendations.py

```diff
@@ -0,0 +1,17 @@
+from typing import Dict, List
+from uuid import UUID
+
+from pydantic import BaseModel, PositiveInt
+
+from poprox_concepts.domain import Article, ClickHistory
+
+
+class RecommendationRequest(BaseModel):
+    todays_articles: List[Article]
+    past_articles: List[Article]
+    click_histories: List[ClickHistory]
+    num_recs: PositiveInt
+
+
+class RecommendationResponse(BaseModel):
```

Comment:
This is one of the things we're going to need to adjust when we want to do images or multiple sections.

I think there are smaller steps before we get to this version, but a plan that's been in my head for representing recommendations like look kind of like this:
```
class RecommendationItem(BaseModel):
    display_type: str,
    data: Article
    
class RecommendationSection(BaseModel):
    display_type: str,
    header: str,
    articles: List[RecommendationItem]

class Recommendation(BaseModel):
    email_subject: str,
    recommendations: List[RecommendationSection]
    
recommendations: Dict[UUID, Recommendation]
```

I don't think I want that for this PR -- But I wanted to type it up quick so we could avoid any changes that would make this particularly tricky. Have we thought about what the process for changing these concepts looks like?

Reply:
For "how do we change these concepts" -- the short-term cheapish solution might involve keeping an "old" and "new" version of the format around for a bit and parsing against a pydantic Union of those two types.